src/lib/core/command.ts

```diff
@@ -0,0 +1,23 @@
+import * as rx from "rxjs";
+import { EventEmitter } from "../../typings/embark";
```

Comment:
We actually don't need typings here and can import `EventEmitter` straight from `lib/core/events` as that type is already defined right there.